wp-content/civi-extensions/civirazorpay/cli/import-settlement-data.php

```
@@ -271,6 +275,8 @@ private function processTransactions(array $transactionsByDay, array &$returnVal
             ->addWhere('is_test', '=', $this->isTest)
             ->addValue($settlementIdField, $settlementId)
             ->addValue($settlementDateField, $settlementDate)
+            ->addValue($feeAmountField, $feeAmount)
```

PHPMD (2.15.0)

Avoid unused local variables such as '$feeAmountField'. (Unused Code Rules)

(UnusedLocalVariable)